### ISSUE TITLE
Creator buttons should be keyboard accessible

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#999]
Creator type menu, field-mode switcher, and "-"/"+" buttons (though Shift-Enter does work for adding new creator rows)

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/76163/keyboard-shortcuts-for-changing-author-tag

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/comment/417401/#Comment_417401

**Comment by dstillman** [comment]
Done for Zotero 6. Will need to be done separately for Zotero 7.

**Comment by tnajdek** [comment]
Done it both branches, we can probably close this issue.
